src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -135,6 +138,16 @@ def execute_command(
         logger.debug(f"Command: {request.command}")
         return self._stub.executeCommand(request, metadata=self._metadata)
 
+    if ansys_version == "24.1.0":
```

Comment:
It should be safe to write `>=` here. I would prefer it if we had a function that we could call for all such queries. Something like:
if `ansys_versions.current >= ansys_versions.version_24_1`:
```suggestion
    if ansys_version >= "24.1.0":
```

Reply:
Done. Thank you.